src/components/EmailNotifications/VerifyEmailAddress.js

```diff
@@ -0,0 +1,251 @@
+import React, { useCallback, useState } from 'react'
+import styled from 'styled-components'
+import {
+  Button,
+  Checkbox,
+  Field,
+  GU,
+  IconCheck,
+  IconCross,
+  Link,
+  TextInput,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import { validateEmail } from '../../utils/notifications-utils'
+import emailIllustration from '../../assets/emailIllustration.svg'
+
+const VerifyEmailAddress = React.memo(function VerifyEmailAddress({
+  updateMode,
+  compactMode,
+  email,
+  onSubscribe,
+  onSubscribeSuccess,
+  onReSendVerificationEmail,
+  onDeleteEmail,
+  onError,
+}) {
+  const theme = useTheme()
+  const [emailAddress, setEmailAddress] = useState('')
+  const [emailInvalid, setEmailInvalid] = useState(null)
+  const [termsAccepted, setTermsAccepted] = useState(false)
+
+  const handleEmailAddressBlur = useCallback(e => {
+    const email = e.target.value
+    setEmailInvalid(!validateEmail(email))
+  }, [])
+
+  const handleEmailAddressChange = useCallback(e => {
+    const email = e.target.value
+    setEmailAddress(email)
+    if (validateEmail(email)) {
+      // Set only as valid while user typing. Use blur to set invalid
+      setEmailInvalid(false)
+    }
+  }, [])
+
+  const handleSubscribe = useCallback(async () => {
+    const { subscribedEmail, error } = await onSubscribe(emailAddress)
+
+    if (error) {
+      onError()
+    }
+    onSubscribeSuccess(subscribedEmail)
+  }, [emailAddress, onError, onSubscribe, onSubscribeSuccess])
+
+  const handleReSendEmail = useCallback(async () => {
+    const { error } = await onReSendVerificationEmail()
+
+    if (error) {
+      onError()
+    }
+  }, [onError, onReSendVerificationEmail])
+
+  return (
+    <div
+      css={`
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        padding-top: ${3 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          display: flex;
+          flex-direction: column;
+          text-align: center;
+        `}
+      >
+        <img src={emailIllustration} />
+        <span
+          css={`
+            ${textStyle('title2')};
+            margin-top: ${4 * GU}px;
+          `}
+        >
+          Verify your email address
+        </span>
+
+        <span
+          css={`
+            ${textStyle('body2')};
+            color: ${theme.surfaceContentSecondary};
+            margin-top: ${1.5 * GU}px;
+          `}
+        >
+          {`Almost there! We’ve sent a verification email to ${email}.
```

Comment:
I think this should be `emailAddress`?

```suggestion
          {`Almost there! We’ve sent a verification email to ${emailAddress}.
```

Reply:
no, is ok like this , this is because this screen has two modes update and the normal one that is the third image. if you take a look to the first image says ` We’ve sent a verification email to ` but says no email (that is because that api endpoint is not working haha) and there is using the juror email detected, and  [here](https://github.com/aragon/court-dashboard/pull/300/files#diff-9485ef7603f80508c2cd1eaf16d62436R48)  i am passing to the parent the new juror email ( i am handling the juror email on the parent since is used across several pages from this flow)